lighty-netconf-device/src/main/java/io/lighty/netconf/device/requests/RpcOutputRequestProcessor.java

```diff
@@ -58,6 +59,10 @@ public RpcDefinition getRpcDefinition() {
         return rpcDefinition;
     }
 
+    public Absolute getRpcDefInputAbsolutePath() {
```

Comment:
Only used by the class children in protected methods - can be protected as well.